bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/tH_tH_x53x53_narrow_MX1100_MH400/tH_tH_x53x53_narrow_MX1100_MH400_customizecards.dat

```diff
@@ -0,0 +1,2 @@
+set param_card mass 6000007 1100
+set param_card mass 6100027 400
```

Comment:
Are you fine with the UFO default widths?

Reply:
I am fine with the default widths!

Reply:
What are the default widths set in your model card?

Reply:
They are set to 1.0 GeV in the UFO